src/main/cpp/dcam/console4.h

```diff
@@ -30,58 +28,53 @@
 // define common macro
 
 #ifndef ASSERT
-#define	ASSERT(c)
+#    define ASSERT(c)
 #endif
 
 // absorb different function
 
 #ifdef WIN32
 
-#if defined(UNICODE) || defined(_UNICODE)
-#define	_T(str)	L##str
-#else
-#define	_T(str)	str
-#endif
+#    if defined(UNICODE) || defined(_UNICODE)
+#        define _T(str) L##str
+#    else
+#        define _T(str) str
+#    endif
 
-#elif defined( MACOSX ) || __ppc64__ || __i386__ || __x86_64__ || defined( LINUX )
+#elif defined(MACOSX) || __ppc64__ || __i386__ || __x86_64__ || defined(LINUX)
 
-#define	_T(str)	str
+#    define _T(str) str
 
 #endif
 
 // absorb Visual Studio 2005 and later
 
-#if ! defined(WIN32) || _MSC_VER < 1400
+#if !defined(WIN32) || _MSC_VER < 1400
```

Comment:
@danielaik Do you know why there is a condition that check if the version of visual studio is before this one?
With recent version the code doesn't compile as `BOOL` (which is defined below) is used in `common.cpp`. Following the comment above, I'm thinking that maybe the condition is inverted.

If I change this, the code compile